backend/dinesmart/views.py

```diff
@@ -0,0 +1,171 @@
+from django.shortcuts import render
+from django.http import HttpResponse, JsonResponse
+from secrets import token_bytes
+import hmac
+import hashlib
+import string
+import time
+import json
+from django.views.decorators.csrf import csrf_exempt
+
+from dinesmart.models import Users, UserAuthTokens, PasswordReset
+
+# Create your views here.
+
+@csrf_exempt
+def create_user(request):
+    #only accept post requests
+    if request.method != "POST":
+        return HttpResponse("only POST calls accepted", status=404)
+
+    #input validation
+    try:
+        payload = json.loads(request.body)
+        email = payload["email"]
+        password = payload["password"]
```

Comment:
Can we hash the passwords? This will save the plaintext password in our User db which is unsafe.

Reply:
Will do in next iteration